Update data apps default secrets (#689)
keboola/connection-docs: components/data-apps/index.md

These secrets will be injected into the `secrets.toml` file upon deployment of the app. 
[Read more about the Streamlit secrets](https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).

## Access Storage from Data App

> By default, the Keboola Streamlit Docker image includes two environment variables, which can be easily used to access keboola storage from your application:

Reviewer: Technicky ty ENVy nejsou soucasti toho image. Pokud bysme pustili image dodany userem (jednoho dne :slightly_smiling_face:), budou tam ty ENVy taky

```suggestion
By default, there are two environment variables, which can be easily used to access keboola storage from your application:
```

Author: aha, super díky. Takže jsi to změnil :-) takto je to tedy ok a můžeme approve?

Reviewer: No ne, to je suggestion :slightly_smiling_face:

Reviewer: Pokud s tim souhlasis, muzu to tam pridat (nebo muzes i sama)
![image](https://github.com/user-attachments/assets/6f23cec2-e456-433b-bbb8-3b0b3e3ad27b)

Reviewer 2: @MonikaFeigler  asi musis data `Commit suggestion`, ja to pak rychle prolitnu

Reviewer 2: commitla jsem to

Author: ajo, pardon, to jsem měla příjmout, díky moc oběma
